sw/device/silicon_creator/lib/otbn_util.c

```diff
@@ -86,7 +87,8 @@ rom_error_t otbn_execute_app(otbn_t *ctx) {
   }
   HARDENED_CHECK_EQ(ctx->app_is_loaded, kHardenedBoolTrue);
 
-  return otbn_execute();
+  rom_error_t err = otbn_execute();
+  SEC_MMIO_WRITE_INCREMENT(kOtbnSecMmioExecute);
```

Comment:
nit: just checked the CI failure and it looks like we are missing a return:
```suggestion
  SEC_MMIO_WRITE_INCREMENT(kOtbnSecMmioExecute);
  return err;
```